### DISCUSSION TITLE
Ability to export specific fields when exporting entries with relationships (e.g. [Name1, Name2] instead of [1, 2]

### DISCUSSION THREAD

**Discussion by shadowcpy** [directus/directus#4639] [Feature Requests]
Hi everyone,
maybe I do not get it, 
but is there a way to specify which contents of a field that is foreign (relationship) get exported when downloading a JSON or CSV file? Because if I change the interface field and display field, this does not affect the printed IDs when exporting foreign fields. I get that the ID is important for keeping the link between collections that have relationships but in some scenarios (e.g printing a list for an annual report, etc..), IDs for these fields like 1 or 2 are not very meaningful and have to be looked up manually when evaluating such a list. Is there a way to set human readable fields to be printed when exporting data yet?

Thanks in advance,
m0rtis0

**Comment by rijkvanzanten** [comment]
Heya! This is already possible in the API by combining the `fields` parameter with the `export` parameter. The app will currently "just" export what is on the root, as you can't select nested-related fields to be shown in the tabular layout yet. That being said, that'll change soon: https://github.com/directus/directus/pull/3592 🙂

**Reply by shadowcpy** [reply]
Okay, thanks! But afaik the PR you mentioned has been rejected and split up with no further progress, and I would propose an even smaller change. Just having the option: "Export as displayed" would be nice, with the export data format being pulled from the already existing "Display Format" options for each field.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
